wiki/osu!_wiki_Contribution_Guide/en.md

```diff
@@ -2,53 +2,53 @@
 
 Thanks for your interest in making the osu! wiki better! At this time, contributing to the osu! wiki will require more time and slightly more effort to make your changes actually happen. This quick-start guide is intended for newcomers that have never worked with GitHub and the osu! wiki contribution workflow before. If you already have experience using GitHub, feel free to skip this guide and follow the common feature-branch workflow for content updates in this repository.
```

Comment:
~`At this time, contributing to the osu! wiki will require more time and slightly more effort to make your changes actually happen.`~

This is worded like it is comparing to something (the old wiki's system) that doesn't exist anymore. You can remove it.